common/djangoapps/student/tests/test_models.py

```diff
@@ -197,3 +197,32 @@ def test_upgrade_deadline_with_schedule_and_professional_mode(self):
         ScheduleFactory(enrollment=enrollment)
         self.assertIsNotNone(enrollment.schedule)
         self.assertIsNone(enrollment.upgrade_deadline)
+
+
+class PendingEmailChangeTests(SharedModuleStoreTestCase):
+    """
+    Tests the deletion of PendingEmailChange records.
+    """
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Why is this a separate method from "setUp"?

Reply:
None of the course or user test objects are affected between different test cases, so we can just set these up once for the whole class.